payment_integration/README.md

```diff
@@ -0,0 +1,3 @@
+### Payment Integration Website
+
+![1](https://user-images.githubusercontent.com/63583646/124602004-0ec97600-de86-11eb-809b-51e0d83a2be6.png)
```

Comment:
Hey, @girija0707! Nice work, just needs a few more changes:

Use the template for your README:

Contents of the README:

Project overview.
Use of the project.
Stack of your project.
How to set up / run your project.
Screenshots of the pages / output.
(A short description under each screenshot... Describing the page a little bit)
This would help visitors / developers in genral.

Reply:
Ok sure I'll do all the required changes 😊